doc/api-extensions.md

```diff
@@ -2340,3 +2340,7 @@ the event as that's usually already covered by the `location` field.
 Adds a new `volatile.uuid` configuration key to all storage volumes, snapshots and buckets.
 This information can be used by storage drivers as a separate identifier besides the name
 when working with volumes.
+
+## `instances_uefi_vars`
+
+This API extension indicates that the `/1.0/instances/{name}/uefi_vars` endpoint exists on the server.
```

Comment:
ah just a minor change here - please can we use `uefi-vars` for the URL as looking at https://documentation.ubuntu.com/lxd/en/latest/api/ we've not used underscore in the URLs previously and have instead used hyphen. Thanks

Reply:
Can you also expand the description of this API extension to explain what it provides, rather than just mentioning the API endpoint exists.

Reply:
Fixed. Thanks!